src/main/java/devices/PowerSupply.java

```diff
@@ -0,0 +1,104 @@
+package devices;
```

Comment:
Provide a [package-info.java](https://stackoverflow.com/questions/624422/how-do-i-document-packages-in-java) file to describe what these packages do.